### ISSUE TITLE
[PT] Update lesson: Ecto Associations

### ISSUE THREAD

**Issue by thiamsantos** [elixirschool/elixirschool#2394]
## Motivation

The portuguese translation for the lesson [Ecto Associations](https://elixirschool.com/en/lessons/ecto/associations/) needs to be updated.
We need to update the version of the lesson from `1.0.1` to `1.2.1`.

## Proposed Solution

Translate the updates of the lesson.

We can run `git diff` to see all differences between the versions of the lesson:

```sh
git diff c1e8f7c60753a3afe4f34d1972631886111ee39f HEAD en/lessons/ecto/associations.md
```

## Tips

- Check the [contributing guidelines](https://github.com/elixirschool/elixirschool/blob/master/CONTRIBUTING.md) before starting.
- Feel free to open a [draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/#tag-your-work-in-progress) as soon as you start working on the translation. In order to prevent we prevent having multiple people working on the translation and also allow the reviewers to give early feedback.
- In case of any doubt feel free to comment in this issue.

**Comment by pablohildo** [comment]
Just worked on this, PR opened!

**Comment by thiamsantos** [comment]
Closed by #2418 

Thanks @pablohildo :tada:
